### PR TITLE
correct avx512_vnni isa name.

### PR DESCRIPTION
**Pull request by xuhancn**
`x86` has two vnni isa currently: `avx2_vnni` and `avx512_vnni`. 
This PR correct the function name to `avx512_vnni`.

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @voznesenskym @penguinwu @EikanWang @Guobing-Chen @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

### CODE REVIEW

**Review by leslie-fang-intel — APPROVED** [review]

File: torch/ao/quantization/qconfig.py [hunk 1]
```diff
@@ -253,7 +253,7 @@ def get_default_qconfig(backend='x86', version=0):
             qconfig = QConfig(activation=HistogramObserver.with_args(reduce_range=False),
                               weight=default_weight_observer)
         elif backend == 'onednn':
-            if not torch.cpu._is_cpu_support_vnni():
+            if not torch.cpu._is_cpu_support_avx512_vnni():
```
Comment (jgong5) [inline torch/ao/quantization/qconfig.py:256]: I guess the caller here actually means "vnni" regardless of whether it is avx2_vnni or avx512_vnni. I would suggest we keep this API unchanged while only fix the implementation to reflect its intention, i.e., either avx2_vnni or avx512_vnni supported means vnni is supported.

**Review by jgong5 — DISMISSED** [review]

File: torch/ao/quantization/qconfig.py (hunk 1 above)
Comment (xuhancn) [reply torch/ao/quantization/qconfig.py]: Okay, let keep using the origian name.

File: torch/cpu/__init__.py [hunk 2]
```diff
@@ -41,7 +41,8 @@ def _is_cpu_support_avx512() -> bool:
 
 def _is_cpu_support_vnni() -> bool:
     r"""Returns a bool indicating if CPU supports VNNI."""
-    return torch._C._cpu._is_cpu_support_vnni()
+    # Note: Currently, it only check avx512_vnni, Maybe need to support avx2_vnni later.
```
Comment (jgong5) [inline torch/cpu/__init__.py:44]:
```suggestion
    # Note: Currently, it only checks avx512_vnni, will add the support of avx2_vnni later.
```

**Review by jgong5 — APPROVED** [review]

**Review by desertfire — APPROVED** [review]

**Pull request closed without merging**